troposphere/constants.py

```diff
@@ -202,3 +202,32 @@
 CACHE_C1_XLARGE = 'cache.c1.xlarge'
 
 CACHE_T1_MICRO = 'cache.t1.micro'
+
+#
+# Parameter types
+#
+STRING = 'String'
```

Comment:
I agree. I prefer using a module, because adding something like _TYPE to all of these will make most of them very long.

I currently use a constants module in my projects, having this as `constants.parameters.types.STRING`. I also have a `constants.parameters.patterns` for common validation patterns (alphanumeric, ...), so depending on which kind of constants you want in troposphere, this could also be `constants.parameter_types`.